README.md

```diff
@@ -7,9 +7,10 @@ This is a quick introduction to get new developers and witnesses up to speed on
  The following dependencies were necessary for a clean install of Ubuntu 18.04 LTS:
 
  ```
- sudo apt-get install gcc-5 g++-5 cmake make libbz2-dev\
-     libdb++-dev libdb-dev libssl-dev openssl libreadline-dev\
-      autoconf libtool git
+ sudo apt-get install autoconf bash build-essential ca-certificates cmake \
+      doxygen git graphviz libbz2-dev libcurl4-openssl-dev libncurses-dev \
+      libreadline-dev libssl-dev libtool libzmq3-dev locales ntp pkg-config \
+      wget
```

Comment:
Are you sure all this dependencies are needed now only to install this pull request ? I was able to build just with `libzmq3-dev`. Of course I already had dependencies for peerplays installed so it was just an upgrade. but i don't see for example `wget` or `libncurses` actually needed.

Reply:
wget is needed fo download boost source code, as instructed in README file.

I do not remember why is libncurses here, but I believe I compiled this list carefully at the time of doing this, and one of the packages was requiring it.

Also, I wanted to have same list of dependencies for docker image and regullar build.